content/06.model-complexity.md

```diff
@@ -11,30 +11,31 @@ All of these contribute to low signal to noise ratio in rare disease datasets.
 Thus, applying ML to rare disease data without addressing the aforementioned shortcomings may lead to models that have low reproducibility or are hard to interpret.
 
 Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (e.g., random forests) (Figure[@fig:3]a).
-Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10/btzfh6]. 
+Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10/btzfh6]. [TODO: consider re-evaluating these references now that we're not explicitly talking about bagging?]
 Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
 Resampling approaches are most helpful in constructing confidence intervals for algorithms that generate the same outcome every time they are run (i.e., deterministic models).
 For decision trees that choose features at random for selecting a path to the outcome (i.e., are non-deterministic), resampling approaches can be helpful in estimating the reproducibility of the model. 
 
-In situations where decision tree-based ensemble methods fail when they are applied to rare disease datasets, cascade learning has been a viable alternative [@doi:10.1007/s11634-019-00354-x; @pmid:30815073].
+In situations where decision tree-based ensemble methods fail when they are applied to rare disease datasets, cascade learning has been a viable alternative [@doi:10.1007/s11634-019-00354-x; @pmid:30815073]. [TODO: first reference is generally about decision trees in imbalanced situations, not about cascade learning per se -- consider moving elsewhere in the sentence?]
```

Comment:
Another question about a reference here.

Reply:
```suggestion
In situations where decision tree-based ensemble methods fail when they are applied to rare disease datasets, cascade learning has been a viable alternative [@pmid:30815073]. [TODO: first reference is generally about decision trees in imbalanced situations, not about cascade learning per se -- consider moving elsewhere in the sentence?]
```

Reply:
moved it to the section above where we talk about imbalanced datasets